### PR TITLE
[Inductor] Enable inductor performance test for CPU

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**
Enable inductor benchmark performance test for CPU. Test scope is aligned with A100 performance test: nightly, weekly and self-defined benchmark test.

### CODE REVIEW

File: .github/workflows/inductor-perf-test-nightly-cpu.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+name: inductor-CPU-perf-nightly
+
+on:
```
Comment (chuanqi129) [inline .github/workflows/inductor-perf-test-nightly-cpu.yml:3]: Could we add cpu nightly test into https://github.com/pytorch/pytorch/blob/main/.github/workflows/inductor-perf-test-nightly.yml also?

File: .ci/pytorch/test.sh [hunk 2]
```diff
@@ -491,6 +491,11 @@ test_perf_for_dashboard() {
         cp "$TEST_REPORTS_DIR/${backend}_with_cudagraphs_${suite}_${dtype}_${mode}_cuda_${target}.csv" \
           "$TEST_REPORTS_DIR/${backend}_cudagraphs_low_precision_${suite}_quant_${mode}_cuda_${target}.csv"
       fi
+      if [[ "$DASHBOARD_TAG" == *inductor_cpu-true* ]]; then
```
Comment (chuanqi129) [inline .ci/pytorch/test.sh:494]: Let's limited the inductor cpu performance nightly start from fp32 and amp inference firstly

File: .github/workflows/inductor-perf-test-nightly-cpu.yml (hunk 1 above)
Comment (WeizhuoZhang-intel) [reply .github/workflows/inductor-perf-test-nightly-cpu.yml:3]: Perhaps not. 
1. `inductor-perf-test-nightly.yml` will run on `linux.gcp.a100` instance. 
2. All the build and test image are cuda images.
3. [Each A100 instance only has 6-core 2.2GHz Xeon CPU](https://pytorch.org/docs/stable/torch.compiler_performance_dashboard.html)

File: .ci/pytorch/test.sh (hunk 2 above)
Comment (WeizhuoZhang-intel) [reply .ci/pytorch/test.sh:494]: Done. I disabled training test. And only run float32 and amp inference for inductor cpu nightly

File: .github/workflows/inductor-perf-test-nightly-cpu.yml (hunk 1 above)
Comment (chuanqi129) [reply .github/workflows/inductor-perf-test-nightly-cpu.yml:3]: We just need to add cpu build and test as new jobs in that workflow

**Comment by chuanqi129** [comment]
Close this one, because @desertfire helped to enable cpu-x86 inductor nightly test by https://github.com/pytorch/pytorch/pull/131369

**Pull request closed without merging**